backend/jupiterbrain.go

```diff
@@ -77,6 +78,8 @@ type jupiterBrainProvider struct {
 
 	imageSelectorType string
 	imageSelector     image.Selector
+	imageCPUs         int
+	imageRAM          int
```

Comment:
Are CPU and RAM allocations a property of the image? The jupiter-brain patch read to me like they were instance-level settings.

FYI I know nothing about this. :)

Reply:
I think the terms "instance" and "image" are roughly interchangeable here, at least the way we use them. The "images" we clone from are just shutdown virtual machines.

Maybe this is needlessly confusing. Would it make more sense if this was something like `defaultInstanceCPUs` and similar for RAM?

Reply:
That makes sense, thanks for explaining! 👍